modules/ballerina-http/src/main/ballerina/ballerina/net/http/httplib.bal

```diff
@@ -0,0 +1,25 @@
+package ballerina.net.http;
+
+const string CONTINUE_100 = "100-continue";
+
+@Description { value:"This function can be used to check whether the client expects a 100-continue response."}
```

Comment:
Shall we remove "This function can be used to" part from the description? IMO, Something like "Checks whether the client expects a 100-continue response or not" will be better

Reply:
Ack.